eman2/protocols/protocol_tomo_boxing.py

```diff
@@ -82,6 +88,45 @@ def _readSetOfCoordinates3D(self, jsonBoxDict, tomo, coord3DSetDict):
 
                 self._readCoordinates3D(box, tomo, coord3DSet)
 
+    def _coordinates2json(self, inputCoor):
```

Comment:
What about moving this method to the convert class? It can potentially be helpful for some other protocols